app/views/content/train-to-be-a-teacher/qualifications-you-need-to-teach.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "What qualifications do I need to be a teacher?"
 heading: "What qualifications do I need to be a teacher?"
```

Comment:
Worth changing this h1 so it is no longer a question? But check re SEO first

Reply:
On GSC 'What qualifications do I need to be a teacher' is the number one search query related to qualifications.

Reply:
Change title but leave H1 for SEO